src/plugins/status_page/public/status_page.html

```diff
@@ -15,27 +15,29 @@
     </div>
   </div>
 
-  <div class="row plugin_status_wrapper">
-    <h3>Installed Plugins</h3>
-    <div ng-if="!ui.statuses && ui.loading" class="loading_statuses">
+  <div class="row statuses_wrapper">
+    <h3>Status Breakdown</h3>
+
+    <div ng-if="!ui.statuses && ui.loading" class="statuses_loading">
       <span class="spinner"></span>
     </div>
 
-    <h4 ng-if="!ui.statuses && !ui.loading" class="missing_statuses">
-      No plugin status information available
+    <h4 ng-if="!ui.statuses && !ui.loading" class="statuses_missing">
+      No status information available
     </h4>
 
-    <table class="plugin_status_breakdown" ng-if="ui.statuses">
+    <table class="statuses" data-test-subj="statusBreakdown" ng-if="ui.statuses">
```

Comment:
Not a blocker for this PR, more for my own curiosity: why did you create the separate `data-test-subj` attribute? Is it something only our tests look for or does it have another purpose?